.dockerignore

```diff
@@ -1,4 +1,6 @@
 bundles
+integration-cli/bundles
```

Comment:
I thought this is already gone via https://github.com/docker/docker/pull/29121 ...